stripe/src/test/java/com/stripe/android/PaymentMethodEndToEndTest.kt

```diff
@@ -214,4 +214,21 @@ class PaymentMethodEndToEndTest {
         assertThat(paymentMethod?.type)
             .isEqualTo(PaymentMethod.Type.Alipay)
     }
+
+    @Test
+    fun createPaymentMethod_withGrabPay_shouldCreateObject() {
+        val repository = StripeApiRepository(
+            context,
+            ApiKeyFixtures.GRABPAY_PUBLISHABLE_KEY,
+            apiVersion = "2020-03-02"
```

Comment:
`"2020-03-02"` is the default API version we use so you shouldn't need to specify it